components/tabs/test/tabs.visual-diff.js

```diff
@@ -118,19 +117,21 @@ describe('d2l-tabs', () => {
 				});
 
 				it('focus next', async function() {
+					await page.keyboard.press('Tab');
```

Comment:
This is the first of several cases where we want to focus on something inside any element's shadow root, which `focusWithKeyboard()` and `focusWithMouse()` don't support.

We have a couple options:
1. This hackery, which essentially duplicates `focusWithKeyboard` but splits it into two. I don't love it.
2. We extend `focusWithKeyboard` to take an optional `shadowRootSelector`. When present, it would add an extra step before focusing to query the first selector's shadow root.

Thoughts?